src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -111,26 +106,6 @@ export default class Mutations extends React.Component<IMutationsPageProps, {}>
 
         return tabs;
     }
-
-    protected bannerAlert(): JSX.Element|null
```

Comment:
`Mutations.tsx` exists only in Results View page, that's why standalone mutation mapper was broken. This part has now been moved into `MutationMapper`